gatling/install-docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+apt-get update
+
+apt-get install -y \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    software-properties-common
+
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
Use `sudo` everywhere or not at all :slightly_smiling_face: 
```suggestion
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
```